test/augmentation/test_augmentation.py

```diff
@@ -145,6 +145,140 @@ def test_gradcheck(self):
         assert gradcheck(RandomHorizontalFlip(p=1.), (input, ), raise_exception=True)
 
 
+class TestRandomVerticalFlip:
+
+    def smoke_test(self):
+        f = RandomVerticalFlip(0.5)
+        repr = "RandomVerticalFlip(p=0.5, return_transform=False)"
+        assert str(f) == repr
+
+    def test_random_vflip(self):
```

Comment:
I miss something  here, tests for cuda. You need to add the following:
```python
from test.common import device

def test_random_vflip(self, device):

input = input.to(device)  # send tensor to the device
```

to verify: `pytest --testtype cuda`